roles/core/precheck/tasks/prepare.yml

```diff
@@ -85,29 +85,29 @@
 #
 # Disable firewall
 #
-- name: prepare | List installed firewall RPMs
-  yum:
-    list: firewalld
-  register: scale_prepare_firewallrpm
-  when: ansible_pkg_mgr == 'yum'
 
-- name: prepare | List installed firewall RPMs
-  dnf:
-    list: firewalld
-  register: scale_prepare_firewalldnfrpm
-  when: ansible_pkg_mgr == 'dnf'
+- block:  # when: scale_prepare_disable_firewall
+    - name: prepare | List installed firewall RPMs
+      yum:
+        list: firewalld
+      register: scale_prepare_firewallpkg
+      when: ansible_pkg_mgr == 'yum'
+
+    - name: prepare | List installed firewall RPMs
+      dnf:
+        list: firewalld
+      register: scale_prepare_firewallpkg
```

Comment:
@acch Do you think same register variable name may create issue. i was facing issue when i was trying to use even in the conditional when condition and its getting overridden.  but not able to remember exactly.  Thanks